solr/core/src/java/org/apache/solr/handler/admin/LoggingHandler.java

```diff
@@ -63,6 +65,9 @@ public void inform(SolrCore core) {
     if (watcher == null) {
       watcher = core.getCoreContainer().getLogging();
     }
+    if (cc == null) {
```

Comment:
does this actually happen (how?).  Otherwise, let's not and furthermore declare cc as final.